File: MedArrival-Backend/src/main/java/com/cnesten/medarrivalbackend/DTO/ClientDTO.java
package com.cnesten.medarrivalbackend.DTO;

import com.cnesten.medarrivalbackend.Models.Client.ClientType;
import lombok.Data;

import java.time.LocalDateTime;

@Data
public class ClientDTO {
    private Long id;
    private String name;
    private String address;
    private ClientType clientType;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private String createdBy;
    private String updatedBy;
}

File: MedArrival-Front/src/models/ClientDTO.ts
import { BaseDTO } from "./BaseDTO";

export interface ClientDTO extends BaseDTO {
    name: string;
    address: string;
    clientType: ClientType;
}

export enum ClientType {
    CLIENT_RP = 'CLIENT_RP',
    CLIENT_MARCHER = 'CLIENT_MARCHER'
  }